experimental/dds/tree2/src/feature-libraries/forestSummarizer.ts

```diff
@@ -112,3 +133,19 @@ export class ForestSummarizer implements Summarizable {
 		}
 	}
 }
+
+function encodeSummary(
```

Comment:
since this is encoding a field not a node, it doesn't, it can't get its top level schema from the cursor. Thus this should take in a fieldSchema.

For the document root it should use the document root schema. 

For other roots (for repair data) it will have to make up some "any" schema.

We can do this by refactoring the logic in laztTree.ts's LazyTree.parentField where it comes up with the schema for a root field out into a function. Put it in typedTreeSchema.ts and take in a fieldKey and TypedSchema collection and return the field schema for that root.

Then you can leave a TODO here to make schemaCompressedEncode use it (which will fix the inefficent encoding I noted on one of the snapshots)

Then you can

Reply:
Added TODO in `schemaCompressedEncode` to use `FieldSchema` to improve the inefficient encoding.